File: cv/car_test.py
#!/usr/bin/env python
# -*- coding:UTF-8 -*-
# ----------------------------------
# File Name : 2.py
# Purpose :
# Creation Date : 16-03-2016
# Last Modified : Wed Mar 16 15:21:25 2016
# Created By : Jeasine Ma
# ---------------------------------

from  move_interface import carHandle as myc

while(True):
    car = myc(0,9600)
    car.send_cmd(0,20)
    car.wait_button(0,'s')

File: CPP/car_test.cpp
/*-----------------------------------------
 File Name : car_test.cpp
 Purpose :
 Creation Date : 04-04-2016
 Last Modified : Mon Apr  4 16:35:21 2016
 Created By : Jeasine Ma
-----------------------------------------*/
#include <iostream>
#include "./catchtest/car_handle.h"

using namespace std;

int main()
{
	CarHandle *car = new CarHandle();
	while(1) {
		car->targetSpeed = 100;
		car->targetAngle = 30;
	}	
}

